caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java

```diff
@@ -170,6 +171,13 @@ public static void main(String[] args) {
                 log.info("authentication using: " + meth);
             }
 
+            final Mode mode;
+            if (am.isSet(MODE_ARG)) {
+                mode = Mode.valueOf(am.getValue(MODE_ARG).toUpperCase());
```

Comment:
If the user supplied value of mode here isn't one of the two (three?) supported ones, then we have to make sure a log.error message is printed with 'unrecognized mode value', usage is printed, and we exist with -1.